src/RVRP_data.jl

```diff
@@ -16,29 +16,39 @@ mutable struct Location # Location where can be a Depot, Pickup, Delivery, Recha
     opening_time_windows::Vector{Range}
     entry_time::Float64
     exit_time::Float64
+    entry_location_group_id::String # to model and extra time when entering/exiting the group
     energy_fixed_cost::Float64 # an entry fee, if any
     energy_unit_cost::Float64 # recharging cost per unit of energy, if any
     energy_recharging_speeds::Vector{Float64} # if recharging in this location: the i-th speep is associted to the i-th energy interval defined for the vehicle
 end
 
-mutable struct LocationGroup # optionally defined to identify a set of locations with some commonalities, such as all possible pickups for a request.
+mutable struct LocationGroup # optionally defined to identify a set of locations with some commonalities, such as all possible pickups for a request, or joint entry/exit times.
     id::String
     location_ids::Vector{String}
+    entry_time::Float64
+    exit_time::Float64
 end
 
-mutable struct ProductCategory # need to be defined only if there are preecedence or conflict restrictions
+mutable struct ProductCategory # To define preecedence or conflict restrictions
     id::String
     conflicting_product_ids::Vector{String} # if any
     prohibited_predecessor_product_ids::Vector{String} # if any
```

Comment:
The two 'product' here mean 'product_category', right?

Reply:
right